faunadb-java-dsl/src/main/java/com/faunadb/client/types/Codec.java

```diff
@@ -12,59 +12,66 @@
 import static java.lang.String.format;
 
 /**
- * Codec is a function that represents an attempt to coerce a {@link Value} to a concrete type.
- * There are pre-defined codecs for each FaunaDB primitive types: {@link Codec#VALUE}, {@link Codec#STRING},
- * {@link Codec#LONG}, {@link Codec#DOUBLE}, {@link Codec#DATE}, {@link Codec#TIME}, {@link Codec#REF},
- * {@link Codec#SET_REF}, {@link Codec#ARRAY}, and {@link Codec#OBJECT}.
- * <p>
- * Codecs return a {@link Result} of the coercion attempt. If it fails to coerce, {@link Result}
- * will contain an error message.
- * <p>
- * It is also possible to create customized codecs to handle complex objects:
+ * Codec instances are used to coerce a {@link Value} to a concrete type.
```

Comment:
I think we should stray away from the word coerce as it has a connotation of being difficult or hard think.   The formal definition of coerce is "persuade (an unwilling person) to do something by using force or threats".  

I would suggested a word like convert, translate, format

Reply:
I're replaced coerce with convert throughout the whole code.